### PR TITLE
Create Mazeedkmk204.json

### PR DESCRIPTION
**Pull request by Mazeedkmk204**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Mazeedkmk204.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Mazeed Khan",
+  "bio": "Open Source DevRel | Frontend Developer | **GitHub** Star",
+  "links": [
+    {
+      "group": "Business",
+      "name": "jabeen taj automobiles",
```
Comment (amandamartin-dev) [inline data/Mazeedkmk204.json:7]: You forgot to add your URL here... like this `"url": "http://examplelink.io",`

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/Mazeedkmk204.json (hunk 1 above)
Comment (amandamartin-dev) [reply data/Mazeedkmk204.json:7]: link to the docs if you need them [Profile with JSON](https://www.biodrop.io/docs/quickstart-json)

**Review by sital002 — COMMENTED** [review]
It looks like you already have created a biodrop profile using forms there is no point in creating it with json again since it will be ignored here is your biodrop profile  using forms https://www.biodrop.io/Mazeedkmk204.
You can close this Pull Request.

**Comment by SaraJaoude** [comment]
> It looks like you already have created a biodrop profile using forms there is no point in creating it with json again since it will be ignored here is your biodrop profile using forms https://www.biodrop.io/Mazeedkmk204. You can close this Pull Request.

Closing this PR for the reasons mentioned above. If you wish to edit your Profile then please check out the[ Docs for Editing through Forms.](https://www.biodrop.io/docs/how-to-guides/editing-forms)

**Pull request closed without merging**